### DISCUSSION TITLE
Welcome to CaptureSight Discussions!

### DISCUSSION THREAD

**Discussion by zaksabeast** [zaksabeast/CaptureSight#66] [General]
## 👋 Welcome!
GitHub discussions are in beta, and CaptureSight seemed like a good little test place to try them out.

Please continue to open issues and feature requests in the [issue tracker](https://github.com/zaksabeast/CaptureSight/issues).

Use [discussions](https://github.com/zaksabeast/CaptureSight/discussions) to:
- Ask questions about how the Pokemon games work or why something in CaptureSight was done a specific way
- Talk about an idea you have for the RNG community
- Share fun stories about CaptureSight usage

Discussions may be disabled in the future, depending on how it goes, but it will be fun to try out for now!

**Comment by jamie0522** [comment]
Question is there any way to check how many advancements are needed to find a shiny alpha in legends arceus?

**Reply by zaksabeast** [reply]
CaptureSight does not have this ability, but I believe [the RNG Discord](https://www.discord.gg/d8JuAvg) has a script in the #legends-arceus channel.

There is also [an open issue](https://github.com/zaksabeast/CaptureSight/issues/88) to add a feature for CaptureSight to show the necessary information in a trade view.

**Comment by Sparx1245** [comment]
Is CaptureSight gonna be compatable with AMS 1.10 in the future or no?
